komfydb/execution/logical_plan/logical_plan.cc

```diff
@@ -4,15 +4,48 @@
 #include <memory>
 #include <string_view>
 
+#include "glog/logging.h"
+
+#include "aggregate_node.h"
 #include "komfydb/common/tuple_desc.h"
+#include "komfydb/execution/aggregate.h"
+#include "komfydb/execution/aggregator.h"
+#include "komfydb/execution/filter.h"
+#include "komfydb/execution/join.h"
+#include "komfydb/execution/project.h"
+#include "komfydb/execution/seq_scan.h"
 #include "komfydb/optimizer/join_optimizer.h"
+#include "komfydb/storage/table_iterator.h"
 #include "komfydb/utils/status_macros.h"
 
 namespace {
 
 using komfydb::common::ColumnRef;
 using komfydb::common::TupleDesc;
 using komfydb::common::Type;
+using komfydb::storage::TableIterator;
+
+// Find (like in Union-Find).
+std::string GetEquiv(std::string& name,
```

Comment:
Isn't `equiv` parameter always the same map? Maybe make this a private function instead of passing it?

Reply:
I think it's about this union-find thing being separete from class logic, but imo it doesn't quite work when you have to keep "equiv" hashmap in the class anyway. In python in logical plan I'd probably keep refrence to the "UnionFind" object which keeps equiv inside and used the interface of it UF in LP. IDK if it makes sense to do it this way here tho

Reply:
Imo this makes the same sense as in Python, but still not sure if it's not a bit overkill.

Reply:
This is a question I cannot answer, i.e. private method vs function in an anonymous namespace. I remember that at Google it was common to have no private methods and only anonymous functions like this, passing only necessary parameters. It is nicer to have less methods in the class declaration so I'll leave it like that.